### ISSUE TITLE
LocationDecisionKS segmentation masks

### ISSUE THREAD

**Issue by ivo--t** [TWOEARS/blackboard-system#46]
I am a bit confused about the changes introduced in edde19f1b0788f2464c9c852c31b0d691ae5bbed and further. It seems the LocationDecisionKS now _always_ also add segmentationHypothesis to the blackboard, without option to prevent this? However usually, the StreamSegregationKS will be used to produce segmentationHypotheses. Since the LocationDecisionKS is supposed to be used always because of its included time-integration, the two will interfere and the SegmentIdentityKS be confused.

Could you clarify / add a switch to LocationDecisionKS's constructor to disable segments production?

**Comment by ningma97** [comment]
No. LocationDecisionKS _only_ adds `segmentationHypothesis` if `TopdownSourceModelKS` is used and produces a mask.
